CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+# Contributing to F#
+
+One of the easiest ways to contribute is to participate in discussions on GitHub issues. You can also contribute by submitting pull requests with code changes.
+
+## General feedback and discussions?
+
+Start a discussion on the [repository issue tracker](https://github.com/dotnet/fsharp/issues).
```

Comment:
```suggestion
Start a [discussion](https://github.com/dotnet/fsharp/discussions) on the [repository issue tracker](https://github.com/dotnet/fsharp/issues).
```
direct link

Reply:
These two contradict, it's either discussion or the issue.